src/webui/api/torrentscontroller.cpp

```diff
@@ -368,6 +415,11 @@ void TorrentsController::infoAction()
 
         QVariantMap serializedTorrent = serialize(*torrent);
 
+        if (includeFiles)
+        {
+            QList<int> fileIndexes;
+            serializedTorrent.insert(KEY_PROP_FILES, getFiles(torrent, fileIndexes));
+        }
```

Comment:
Taking into account the changes proposed in https://github.com/qbittorrent/qBittorrent/pull/22750/files#r2103131571:
```suggestion
        if (includeFiles)
            serializedTorrent.insert(KEY_PROP_FILES, getFiles(torrent));
```